### PR TITLE
Hot-fix coming through!

### PR DESCRIPTION
**Pull request by dambem**
## Relevant issue
So, let's talk a little bit about what I did.

Basically, here's the deal. The URL changed from luftdaten to Archive.sensor.community, which is also on a faster server and should now work a lot smoother! Brilliant, everything's great isn't it, all lovely and nothings broken.

Well no. So what was happening on my end was the front-end kept breaking. I couldn't understand why, and I thought oh it must be due to the new website having too much data. Nope. Javascript is actually really good at parsing this data it turns out. What I had done was begin implementing some fun changes in this branch. One of those changes consisted of a fractional reducer. Essentially what it would do is reduce the human_display to show the smallest possible fraction. (e.g, if 10 out of 100 people would die due to pollution, it would reduce it to 1/10)

However, it's been a very nice day over the past 24 hours, so what happened was that 0.96 people out of 500 would die due to pollution. This was not a number that the fractional reducer for some reason can deal with, and decided to blow up in my face, as it couldn't reduce that fraction to whole numbers.

Basically, it's fixed don't worry about it accept this, merge it and move on. 
## Checklist
- [ ] Tests all pass

### CODE REVIEW

**Comment by Megan-Wright** [comment]
@damben just tested this out and still getting absolutely no sensor data :/

**Comment by dambem** [comment]
Arghh okay hang on getting it sorted.

**Comment by dambem** [comment]
In the meantime, can somebody open up an issue for a local backup? I should be able to sort it out but just want to get this stuff ready!

**Comment by dambem** [comment]
@Megan-Wright Can you try this one for now and let me know?

**Comment by Megan-Wright** [comment]
@dambem still nothing :(

**Comment by dambem** [comment]
It had turned out they deleted them temporarily, this works for me, just needs a little time to allow them to pop through https://github.com/opendata-stuttgart/meta/issues/148

**Review by Megan-Wright — APPROVED** [review]
Ok that's taking at least a full minute to come through on initial start up so should probably look into that, but it does in fact work - approved!
